documentation/Windows-Installation-Guide.md

```diff
@@ -0,0 +1,399 @@
+![JRE Installation Folder](https://user-images.githubusercontent.com/19670372/33494871-7e2cadf8-d6bc-11e7-84fc-10864c159c69.png)
+- Click Finish on the next screen, once the installation is completed.
+
+##### Check Installation 
+
+- Go to the command prompt and type 'java -version'
+![Java not Installed](https://user-images.githubusercontent.com/19670372/33494868-7de2792c-d6bc-11e7-81b1-5b9726168cf4.png)
+- If you see something like above then Environment Variable is not set.
+
+##### Set Environment Variable
+
+- Open File explorer -> Right click on 'This PC' -> Click on Properties, Once you do that you will get a window like 
+this  
+![Advanced System Setting](https://user-images.githubusercontent.com/19670372/33494861-7d4fbad8-d6bc-11e7-8d7e-28b299621a11.png)
+- Click on **Advanced System Settings**. You should get a Window like this
+![System Properties](https://user-images.githubusercontent.com/19670372/33494867-7dc95e9c-d6bc-11e7-8462-767075c59c74.png)
+- Click on **Environment Variables**, which should give you a window like this
+![Environment Variables](https://user-images.githubusercontent.com/19670372/33494880-7ef94e94-d6bc-11e7-8743-a3aee641e77d.png)
+- Click on **Path** and Click **Edit**. In **Variable value** at the end type the path of the Java installation directory, 
+for which you made the note earlier which in my case is ```C:\Program Files\Java\jdk1.8.0_151\``` and append **bin** at the 
+end like this.  
+![Edit User Variable](https://user-images.githubusercontent.com/19670372/33494865-7d9ac64a-d6bc-11e7-8e5d-534d1e7bd691.png)
+- Click OK and open a new **Command Prompt** window (as explained earlier) and type **java -version**. 
+Now you should see something like this.  
+![Java Installed](https://user-images.githubusercontent.com/19670372/33494869-7dfd288a-d6bc-11e7-8aaa-5ae94faf4ca8.png)
+
+###### Congratulations you have successfully installed java, now let's move on to installing Gradle 
+
+
+## Installing Gradle
+
+### Check if gradle is installed
+
+- Open **Command Prompt** and type **gradle --version**. If your output looks like this
+![Gradle not Installed](https://user-images.githubusercontent.com/19670372/33494881-7f11dff4-d6bc-11e7-838e-78788c9f6bef.png)
+- Then the gradle application is not installed.
+
+### Download and Install Gradle
+
+- In order to install gradle go [here](https://gradle.org/releases/)  and scroll down for the latest version and click 
+on complete as shown here
+![Gradle Download](https://user-images.githubusercontent.com/19670372/33494883-7f4a4d4e-d6bc-11e7-81ed-a87841741f9c.png)
+- Save the file and unzip it at your preferred location, in my case it is unzipped under **Program Files** in order to be 
+consistent as application are usually installed under **Program Files**.
+- Once unzipped navigate to the bin folder of gradle which in my case is 
+```C:\Program Files\gradle-4.3.1-all\gradle-4.3.1\bin``` as the files were extracted in **Program Files** as described 
```

Comment:
I think for inline code a single backtick is recommended. The 3 backticks are usually for a code block and need to be on their own lines.

Reply:
Not a problem on the PR, someone else can tidy this up in the future 😉